libcxx/test/benchmarks/containers/flat_map.bench.cpp

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
```

Comment:
```suggestion
// REQUIRES: std-at-least-c++26
```
This patch has landed :-)

Reply:
It's at least more idiomatic, maybe some day in the future we feel different.
This comment was indeed mainly about this place; these lines really convinced me this is not scalable.